app/views/salons/new.html.erb

```diff
@@ -0,0 +1,15 @@
+Enter your Salon details
```

Comment:
shouldn't it be index and not new

Reply:
if you want to give option to create new salon in listing

Still base action should be index, with form for new salon included there and no new action required